### PR TITLE
Update readme + Hot fixes

### PR DESCRIPTION
**Pull request by corentingiraud**
**What's the problem this PR addresses?**

- The readme was not up to date and contained some mistakes.
- *FRONT*: Some runtime error with the [date-fns](https://github.com/date-fns/date-fns) lib.
- *BACK*: Sql error in a query because of the use of a `HAVING` clause without a `GROUP BY`

**How did you fix it?**

- Rewrite the readme
- Convert properties to `Date` object using `new Date()`
- Add `GROUP BY` clause.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,10 +150,40 @@ Then: `docker-compose up`
...
+* For [Webstorm](http://plugins.jetbrains.com/plugin/7494)
+
+#### Hot reload
+
+- `Backend`
+The app uses [nodemon](https://nodemon.io/) to watch for code change.
+The app will restart or reload when you edit the code.
+- `Frontend`
+The app uses `angular-cli` to watch for code change.
+The app will restart or reload when you edit the code.
```
Comment (m-germain) [inline README.md]: Should we add how to verify the email when creating acount on the App on dev mode ?

File: README.md [hunk 2]
```diff
@@ -43,92 +60,87 @@ After installing NodeJS,
...
 P.S.: The file `proxy.conf.json` is already ignored by git.
 
+##### Create the super admin account
 
-### Environment variables
-
-Environment variables are parsed with [nconf](https://github.com/indexzero/nconf/).
-The separator used is `__` and words are transformed into camelCase.
+We provide a simple _cli_ to perform some automated tasks (more information [here](#cli)).
+To create the super admin account, use the following command from the root folder of the project:
```
Comment (m-germain) [inline README.md]: Its on the read me twice no ? in line 135 to 140 ? ;)

File: packages/client/src/app/home/home-page/home-page.component.ts [hunk 3]
```diff
@@ -13,6 +13,7 @@ export class HomePageComponent implements OnInit {
 
   ngOnInit(): void {
     this.authService.$currentUser.subscribe((curUser) => {
+      curUser.createdAt = new Date(curUser.createdAt);
```
Comment (Embraser01) [inline packages/client/src/app/home/home-page/home-page.component.ts]: These changes will not be needed when the migration to moleculer will be over as Dates will be automaticaly parsed through a Request Interceptor

File: packages/server/app/services/barman-service.js [hunk 4]
```diff
@@ -358,6 +358,7 @@ async function createServiceBarman(barmanId, servicesId) {
         [Op.gt]: Sequelize.fn('count', Sequelize.col('barmen.id')),
       },
     },
+    group: [Sequelize.col('barmen.id')],
```
Comment (Embraser01) [inline packages/server/app/services/barman-service.js]: Does this have any impact on the front?

**Pull request closed without merging**